docs/content/integrations/airflow/migrating-to-dagster.mdx

```diff
@@ -0,0 +1,92 @@
+
+migrated_airflow_repo = make_dagster_repo_from_airflow_dags_path(
+    os.path.join(os.getenv("AIRFLOW_HOME"), "dags"),
+    "migrated_airflow_repo",
+)
+```
+
+Under the hood, dagster is still running the exact operator code as you were in Airflow. You will be able to view your normal airflow stdout/stderr logs as compute logs in dagit.
+
+---
+
+## Step 2: Define your Airflow connections
+
+By default each Job Run of your migrated DAGs will create an ephemeral airflow metadatabase scoped to each Job run, this means any [Airflow Connections](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html) that your DAG was depending on will need to be created for that to work, to do this you can provide a `connections` parameter to <PyObject module="dagster_airflow" object="make_dagster_repo_from_airflow_dags_path" />
+
+```python file=/integrations/airflow/migrate_repo_connections.py
+import os
+
+from airflow.models import Connection
+from dagster_airflow import make_dagster_repo_from_airflow_dags_path
+
+migrated_airflow_repo = make_dagster_repo_from_airflow_dags_path(
+    os.path.join(os.getenv("AIRFLOW_HOME"), "dags"),
+    "migrated_airflow_repo",
+    connections=[
+        Connection(conn_id="http_default", conn_type="uri", host="https://google.com")
+    ],
+)
+```
+
+If you're running on Dagster Cloud and any of your connection fields are sensitive you can also pass them from [Environment Variables](/dagster-cloud/developing-testing/environment-variables-and-secrets) instead of hard coding them.
+
+---
+
+## Step 3: Run your migrated DAGs locally
+
+Once we have a Repository defined we can launch dagit, view your newly migrated Jobs and launch runs.
+
+```bash
+dagit -f <your migrated dags>.py
+```
+
+If your local shell has the correct permissions you can also test out running the migrated dags end to end.
+
+---
+
+## Step 4: Setup your permissions
+
+Your airflow instance likely had specific IAM or Kubernetes permissions that allowed it to successfully run your airflow dags, in order to run the migrated dagster jobs you'll need to duplicate these permissions dagster.
+
+In general if you can lean on [Airflow Connections](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html) or [Environment Variables](/dagster-cloud/developing-testing/environment-variables-and-secrets) for defining these permissions it will be easier than modifying the underlying dagster infrastructure to assume roles that are permissioned correctly.
+
+### KubernetesPodOperators
+
+If your airflow dags were using [KubernetesPodOperators](https://airflow.apache.org/docs/apache-airflow-providers-cncf-kubernetes/stable/operators.html) its possible that you were using the `in_cluster` config or loading a kube_config file. When migrating switching to using a [Connection with a kube_config JSON blob](https://airflow.apache.org/docs/apache-airflow-providers-cncf-kubernetes/stable/connections/kubernetes.html) will be easier.
```

Comment:
```suggestion
- **Using [Airflow connections](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html) or [environment variables](/dagster-cloud/developing-testing/environment-variables-and-secrets)** are the easiest methods for defining permissions.

- **If your Airflow DAGs used [`KubernetesPodOperators`](https://airflow.apache.org/docs/apache-airflow-providers-cncf-kubernetes/stable/operators.html)**, it's possible that you used the `in_cluster` config or loading a `kube_config` file. When migrating, we recommend switching to [using connections with a `kube_config` JSON blob](https://airflow.apache.org/docs/apache-airflow-providers-cncf-kubernetes/stable/connections/kubernetes.html) to make things easier.
```

Having the Kube stuff be its own section felt a little odd, so I'm suggesting removing the heading and putting these options into a list instead.